CMakeLists.txt

```diff
@@ -22,6 +22,8 @@ if(NOT CMAKE_C_COMPILER_ID MATCHES "^(Intel|GNU|Clang|AppleClang)$")
   message(WARNING "Compiler not officially supported: ${CMAKE_C_COMPILER_ID}")
 endif()
 
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

Comment:
Why are we explicitly setting this flag?

Reply:
What is the default value when CMake is building static libraries?

Reply:
This has come up on a number of occasions  

Rahul and JEDI:
https://github.com/NOAA-EMC/NCEPLIBS/issues/67#issuecomment-666430348

Building on Ubuntu, which makes executables PIE by default:
https://github.com/NOAA-EMC/NCEPLIBS/issues/91

Or hypothetically something like Python bindings. Maybe not for this particular library, but when building wgrib2 for use with Python I needed linked libraries to be PIC.

Reply:
OFF (False)